modules/developing/pages/astream-functions.adoc

```diff
@@ -287,7 +287,7 @@ See <<controlling-your-function,Controlling your function>> for more information
 
 == Deploy functions in the {astra-ui}
 
-. In the {astra-ui} navigation menu, click *Streaming*, and then select a tenant.
+. In the {astra-ui} navigation menu, click *Streaming*, and then click the *Name* of a tenant.
```

Comment:
I'm not sure if Name should be bold here since (I think?) there isn't the literal text "Name", but I don't have a strong opinion on this

Reply:
It's referring to the Name column, but maybe I should just leave it normal text.

Reply:
You're right the bold doesn't make sense here. I was overthinking it. Thanks!